src/pickup_meta.rs.in

```diff
@@ -1,11 +1,573 @@
 // This file is generated by bin/resource_tracing.rs
 
 
-pub const PICKUP_LOCATIONS: &[(&str, &[RoomInfo]); 5] = &[
+pub const PICKUP_LOCATIONS: &[(&str, &[RoomInfo]); 7] = &[
```

Comment:
Maybe this should be renamed since it now describes more than just the pickups. `ROOM_INFO` maybe?